modules/services/cloudwatch/sns.tf

```diff
@@ -0,0 +1,32 @@
+## Default actions which can be overridden using input variables if so desired
+locals {
+  opencraft_ops_email = "ops@opencraft.com"
+  opencraft_urgent_email = "urgent@opencraft.com"
+}
```

Comment:
Instead of hardcoding the email ids, we should supply them as variables, probably a list of email ids?

Reply:
removed